tests/panels/test_sql.py

```diff
@@ -122,6 +122,18 @@ def test_param_conversion(self):
             '["2017-12-22 16:07:01"]'
         ))
 
+    @unittest.skipUnless(connection.vendor not in ('sqlite', 'postgresql'), '')
```

Comment:
I think avoiding the double negative would be more readble.

I'm a little confused as to why these DBs are excluded. How do they handle `BinaryField`?

Reply:
### SQLite:

```
Traceback (most recent call last):
  File "tests/panels/test_sql.py", line 130, in test_binary_param_force_text
    cursor.execute("SELECT * FROM auth_user WHERE username = %s", [b'\xff'])

ProgrammingError: You must not use 8-bit bytestrings unless you use a text_factory that can interpret 8-bit bytestrings (like text_factory = str). It is highly recommended that you instead just switch your application to Unicode strings.
```
### Postgresql:

```
Traceback (most recent call last):
  File "/home/travis/build/jazzband/django-debug-toolbar/.tox/postgresql/lib/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
psycopg2.ProgrammingError: operator does not exist: character varying = bytea
LINE 1: SELECT * FROM auth_user WHERE username = '\xff'::bytea
                                               ^
HINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.
```

Reply:
I'm suggesting that you write this as:

```py
@unittest.skipIf(connection.vendor in ('sqlite', 'postgresql'), '')
```

You should also fill in the reason string instead of using the empty string.

Reply:
Yes, I understand your suggestion, I will make the change when I get a chance.
I was just answering the question you had asked about why they were being excluded.

Reply:
Can you suggest a reason I could place in the `skipIf`?

Reply:
Maybe something like:

```
'Mixing bytestrings and text is not allowed on PostgreSQL and SQLite'
```

WDYT? Feel free to tweak it.